graylog2-server/src/main/java/org/graylog2/utilities/ConfigurationMapConverter.java

```diff
@@ -62,6 +65,13 @@
                 case "boolean":
                     value = "true".equalsIgnoreCase(String.valueOf(entry.getValue()));
                     break;
+                case "list":
+                    final List<?> valueList = entry.getValue() == null ? Collections.emptyList() : (List<?>) entry.getValue();
```

Comment:
No, at that point the type of the elements inside the list is still unknown. The elements are converted into strings in the stream in the next line, and stored in the `value` variable, that will be then stored in the effective configuration map.

I think this is sensible at this point, as this method is used to convert data provided by the user, and it's always better being a bit more cautious about it to avoid surprises.